### ISSUE TITLE
Discord invite is invalid / no way to join discord server

### ISSUE THREAD

**Issue by jackklika** [astral-sh/ruff#14004]
Hi, I can't find a way to join the discord channel. The current invite link on the README of this project and the link in the docs links to an "invite invalid" screen.

I also cannot find the channel when searching `astral` `astral-sh` or `ruff` on the discord "explore channels" screen.

**Comment by MichaReiser** [comment]
Thanks for reporting this. Hmm, I just clicked on the link and it correctly opened [Astral's discord](https://discord.com/invite/astral-sh). Can you share the link that you clicked? Just to make sure we're talking bout the same link

**Comment by jackklika** [comment]
I just clicked it and see the following:
![image](https://github.com/user-attachments/assets/4b0d9a12-2c60-41c9-be1f-cd87f9381d11)

Looking at the dev tools I'm getting the following 403 so it is probably my account...
```
{
    "message": "You need to verify your account in order to perform this action.",
    "code": 40002
}
```
I got in by opening incognito window and logging in there. Not sure why my current session was unverified. 

This can be closed, thanks for your patience.
